automated/bin/Archive_Concordance.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+set -u
+
+# identify on which cluster this script will be executed
+HOST=$(hostname -s)
+
+# source config files
+
+source ${EBROOTGAP}/automated/etc/${HOST}.cfg
+
+#Checking if VCF File array is older then ArchiveTime and archive file if this is the case
+
+for i in $(find "/groups/umcg-gd/${TMP_LFS}/Concordance/array/" -type f -maxdepth 1 -mtime +31 -print)
```

Comment:
Is het necessary to use the hard-coded group name or can this also be fetched from a config file?

Reply:
I think the issue is that this is executed by the gap-ateambot user, you can of course put in a configfile ;)